online-shop/src/app/app.component.html

```diff
@@ -0,0 +1,12 @@
+
+<header>
+  <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/ac/Logo_eMAG.svg/1280px-Logo_eMAG.svg.png"
```

Comment:
**suggestion:** try to have images as actual assets in your codebase (inside the `/assets` folder)